test/test_sorter.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+from src import sorter
+
+
+class validateTester(unittest.TestCase):
+    def test(self):  
+        self.assertTrue(sorter.getSort([3, 3, 5, 4, 10, 11, 12]) == [3, 3, 4, 5, 10, 11, 12])
```

Comment:
Sorting means all of the values must be in memory before processing. What if it's a streaming use-case? Is there a way to solve this without sorting?